src/cprover/bv_pointers_wide.cpp

```diff
@@ -402,7 +403,7 @@ bvt bv_pointers_widet::convert_pointer_type(const exprt &expr)
 
     bvt bv;
 
-    mp_integer size = 0;
+    bytest size{1};
```

Comment:
This changed the value, along with the type - is this correct?

Reply:
Fair point: this was perhaps not intended, but actually doesn't matter for it is necessarily overwritten. So I am now changing the logic to make this an `optionalt<bytest>` instead.